src/Core/Prefab/Globe/Atmosphere.js

```diff
@@ -15,186 +17,219 @@ import groundVS from './Shaders/groundVS.glsl';
 import GlowFS from './Shaders/GlowFS.glsl';
 import GlowVS from './Shaders/GlowVS.glsl';
 
-export const LIGHTING_POSITION = new THREE.Vector3(1, 0, 0);
+const LIGHTING_POSITION = new THREE.Vector3(1, 0, 0);
+const v = new THREE.Vector3();
+
+const coordCam = new C.EPSG_4326();
```

Comment:
This made sense to have all those constants in `GlobeView`, because this atmosphere could defer if it isn't applied on the classic ellipsoid, but then again we don't have any other case for now, so I'm OK with it.

Reply:
Yes, for the moment, geometries used in atmosphere, are ellipsoidal.